src/main/java/org/aksw/gerbil/dataset/impl/generic/GenericDataset.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This file is part of General Entity Annotator Benchmark.
+ *
+ * General Entity Annotator Benchmark is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * General Entity Annotator Benchmark is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with General Entity Annotator Benchmark.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.aksw.gerbil.dataset.impl.generic;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.aksw.gerbil.dataset.InitializableDataset;
+import org.aksw.gerbil.dataset.impl.AbstractDataset;
+import org.aksw.gerbil.datatypes.ErrorTypes;
+import org.aksw.gerbil.exceptions.GerbilException;
+import org.aksw.gerbil.transfer.nif.Document;
+
+public class GenericDataset extends AbstractDataset implements InitializableDataset {
```

Comment:
Shouldn't it have a name like `GenericXMLDataset` or something similar?

Btw. a short javadoc description of the class would be good :wink: